assets/src/components/ladder.tsx

```diff
@@ -39,38 +45,70 @@ const Ladder = ({
   ladderDirection,
   selectedVehicleId,
 }: Props) => {
-  const dispatch = useContext(DispatchContext)
-  const timepointSpacingY =
-    (HEIGHT - MARGIN_TOP_BOTTOM * 2) / (timepoints.length - 1)
-
   const orderedTimepoints: Timepoint[] =
     // Use slice to make a copy of the array before destructively reversing
     ladderDirection === LadderDirection.OneToZero
       ? timepoints.slice().reverse()
       : timepoints
 
+  const timepointSpacingY: TimepointSpacingYFunc = (height: number): number =>
+    (height - MARGIN_TOP_BOTTOM * 2) / (timepoints.length - 1)
   const timepointStatusY = timepointStatusYFromTimepoints(
     timepoints,
     timepointSpacingY
   )
+
   const { ladderVehicles, widthOfLanes } = ladderVehiclesFromVehicles(
     vehicles,
     ladderDirection,
     timepointStatusY
   )
 
   const width = 120 + 2 * widthOfLanes
-  // (0, 0) is in the center of the first timepoint
-  const viewBox = [-width / 2, -MARGIN_TOP_BOTTOM, width, HEIGHT].join(" ")
 
   return (
-    <svg className="m-ladder" height={HEIGHT} width={width} viewBox={viewBox}>
+    <div className="m-ladder">
+      <LadderSvg
+        timepoints={orderedTimepoints}
+        ladderVehicles={ladderVehicles}
+        selectedVehicleId={selectedVehicleId}
+        width={width}
+        timepointSpacingY={timepointSpacingY}
+        timepointStatusY={timepointStatusY}
+      />
+    </div>
+  )
+}
+
+const LadderSvg = ({
+  timepoints,
+  ladderVehicles,
+  selectedVehicleId,
+  width,
+  timepointSpacingY,
+  timepointStatusY,
+}: {
+  timepoints: Timepoint[]
+  ladderVehicles: LadderVehicle[]
+  selectedVehicleId?: VehicleId
+  width: number
+  timepointSpacingY: TimepointSpacingYFunc
+  timepointStatusY: TimepointStatusYFunc
+}) => {
+  const dispatch = useContext(DispatchContext)
+  const [height, setHeight] = useState(500)
```

Comment:
Is the the same 500 as `ESTIMATED_HEIGHT`?

Reply:
It technically doesn't have to be, but it's the same gist so I switched to that.